### PR TITLE
Update readme with the current state of testing.

### PR DESCRIPTION
**Pull request by DailyDreaming**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,50 @@ We welcome feedback and suggestions from any Biodata Catalyst team on useful tes
...
+ - https://firecloud-orchestration.dsde-alpha.broadinstitute.org/status
+ - https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/importPFB
+ - https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/importPFB/{job_id}
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/submissions
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/methodconfigs
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/methodconfigs?allRepos=true
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/methodconfigs/{billing_project}/{workflow}
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}/submissions/{submission_id}
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces
+ - https://rawls.dsde-alpha.broadinstitute.org/api/workspaces/{billing_project}/{workspace}
```
Comment (xbrianh) [inline README.md:97]: Are any Gen3 endpoints tested?

File: README.md [hunk 2]
```diff
@@ -48,3 +48,50 @@ We welcome feedback and suggestions from any Biodata Catalyst team on useful tes
...
+| Terra         | :heavy_multiplication_x: | :heavy_check_mark:       | :heavy_check_mark:       | :heavy_multiplication_x: | Future                   |      |
+| Gen3          | TBD                      | :heavy_multiplication_x: | TBD?                     | TBD?                     |                          |      |
+| Dockstore     |                          |                          | :heavy_multiplication_x: |                          |                          |      |
+| Seven Bridges | :heavy_multiplication_x: | TBD?                     | TBD?                     | :heavy_multiplication_x: | Future                   |      |
+| PIC-SURE      |                          | TBD?                     |                          |                          | :heavy_multiplication_x: |      |
+| HeLx          | TBD?                     | TBD?                     |                          | TBD?                     |                          | :heavy_multiplication_x: |
+
+ - :heavy_check_mark: : Implemented (probably not comprehensively; additional testing suggestions are very welcome)
+ - :heavy_multiplication_x: : Not Applicable
```
Comment (xbrianh) [inline README.md:63]: Consider three indicators:
  - (x) no integrations tested
  - (foo) some integrations tested
  - (check) all integrations tested

**Review by xbrianh — APPROVED** [review]
looks good to me overall, see comments

File: README.md (hunk 1 above)
Comment (DailyDreaming) [reply README.md:97]: No, there were and credentialing is set up, but we only go from Terra to Gen3 and I'd like to see those tests if possible too.

File: README.md (hunk 2 above)
Comment (DailyDreaming) [reply README.md:63]: After speaking earlier, will implement :white_check_mark: for partial, though "all integrations tested" is unlikely to ever happen I think (similar to 100% coverage being a rare beast indeed).

**Pull request merged**